nexus/db-model/src/schema.rs

```diff
@@ -712,6 +712,13 @@ table! {
     }
 }
 
+table! {
+    instance_ssk_key (instance_id, ssh_key_id) {
```

Comment:
```suggestion
    instance_ssh_key (instance_id, ssh_key_id) {
```

(unless I'm missing something and this is deliberate)

Reply:
that was not deliberate! Thanks for the catch